modules/ide-integration/README.md

```diff
@@ -62,6 +62,10 @@ Using this module, you can setup a full Airavata installation inside Intelij IDE
 ### Starting API Server
 
 * Go to org.apache.airavata.ide.integration.APIServerStarter class and right click on the editor and click Run option. This will start Airavata server
+* Make sure that the IDE's working directory for APIServerStarter points to:
+```
+PATH/TO/airavata/modules/ide-integration/src/main/resources
```

Comment:
Change to <Airavata base directory>/modules/ide-integration/src/main/resources

Reply:
fixed!